File: Data/Project Classes/PairDetails.cs
namespace SJC_Portal.Data
{
    public class PairDetails
    {
        public string? Title { get; set; }
        public double? Value { get; set; }
        public double? Percent {  get; set; }
        public double? SecondValue { get; set; }
    }

    public class PairList
    {
        public IEnumerable<PairDetails>? Pairs { get; set; }
    }
}

File: wwwroot/js/ProjectClass/PairDetails.js
export class PairList {
    Pairs;
    constructor() {
        this.Pairs = [new PairDetails()];
        this.Pairs.pop();
    }
}

export class PairDetails {
    Title;
    Value;
    Percent;
    SecondValue;
    constructor() {
        this.Title = "";
        this.Value = 0;
        this.Percent = 0;
        this.SecondValue = 0;
    }
}